llama_stack/apis/inference/inference.py

```diff
@@ -74,11 +74,35 @@ class ChatCompletionResponseEvent(BaseModel):
     stop_reason: Optional[StopReason] = None
 
 
+class ResponseFormatType(Enum):
+    json_schema = "json_schema"
+    grammar = "grammar"
```

Comment:
what does grammar mean here?

Reply:
really means EBNF (as the corresponding struct below has in the payload). maybe I should call it that?